components/boxLink/BoxLink.js

```diff
@@ -0,0 +1,52 @@
+import { getString } from '../../lib/richText';
+import {
+  BoxLinkContainer,
+  EllipseContainer,
+  Ellipse,
+  Title,
+  Description,
+  MediumArrow,
+  LongArrow,
+  InnerContentContainer,
+} from './BoxLink.styles';
+import { icons } from '../../style/icons';
+import PageLink from '../pageLink/PageLink';
+import { breakpointsObj } from '../../lib/responsive';
+import MediaQuery from 'react-responsive';
+export default function BoxLink({ boxLinkData }) {
```

Comment:
formatting: can you add an empty line between imports and `export default function BoxLink...`